editor/svgutils.js

```diff
@@ -529,19 +529,387 @@ svgedit.utilities.getBBox = function(elem) {
 	return ret;
 };
 
-// Function: svgedit.utilities.getRotationAngle
-// Get the rotation angle of the given/selected DOM element
+svgedit.utilities.getPathDFromSegments = function( pathSegments) {
```

Comment:
This is the 4th time I've seen a leading space in front of the first argument, so maybe it's a coding convention I'm not familiar with?